### PR TITLE
Pin jrjackson and update logstash

### PR DESCRIPTION
**Pull request by pxlpnk**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -9,7 +9,9 @@ group :test do
   # logstash does not release any gems on rubygems, but they have two gemspecs within their repo.
   # Using the tag is an attempt of having a stable version to test against where we can ensure that
   # we test against the correct code.
-  gem 'logstash-event', github: 'elastic/logstash', tag: 'v1.5.0'
+  gem 'logstash-event', github: 'elastic/logstash', tag: 'v1.5.4'
+  # logstash 1.5.4 is only supported with jrjackson up to  0.2.9
+  gem 'jrjackson', '~> 0.2.9', :platforms => :jruby
```
Comment (benlovell) [inline Gemfile]: This should be pinned at `0.2.9` instead as we don't commit the lock file.

**Comment by benlovell** [comment]
Nice work on picking this one up :+1: 

Don't forget to do the same in the other gemfiles for the testing matrix.

File: Gemfile (hunk 1 above)
Comment (benlovell) [inline Gemfile]: Also, use the 1.9 style hash syntax.

Comment (pxlpnk) [inline Gemfile]: ah copy and paste made me do it :1234: 

you are right it should be pinned to 0.2.9.

**Comment by pxlpnk** [comment]
Updated it.

**Comment by benlovell** [comment]
❤️

**Pull request merged**